server/game/cards/plots/06/duel.js

```diff
@@ -0,0 +1,65 @@
+const PlotCard = require('../../../plotcard.js');
+
+class Duel extends PlotCard {
+    setupCardAbilities() {
+        this.whenRevealed({
+            handler: () => {
+                let opponent = this.game.getOtherPlayer(this.controller);
+                if(!opponent) {
+                    return;
+                }
+
+                if(this.notEnoughTargets()) {
+                    return;
+                }
+
+                this.game.addMessage('{0} uses {1} to have {2} choose 2 non-Army characters with printed cost 6 or higher', 
+                                      this.controller, this, opponent);
+
+                this.game.promptForSelect(opponent, {
+                    numCards: 2,
+                    activePromptTitle: 'Select two characters',
+                    source: this,
+                    cardCondition: card => !card.hasTrait('Army') && card.getType() === 'character' && card.getCost() >= 6,
+                    onSelect: (player, cards) => this.targetsSelected(player, cards)
+                });
+            }
+        });
+    }
+
+    targetsSelected(player, cards) {
+        this.targets = cards;
+        this.game.addMessage('{0} has chosen {1} as the targets for {2}', player, cards, this);
+
+        this.game.promptWithMenu(player, this, {
+            activePrompt: {
+                menuTitle: 'Choose character to kneel',
+                buttons: [
+                    { text: cards[0].name, method: 'resolve', arg: [0,1]},
```

Comment:
This is pretty clever use of arg. If we're going to do a menu here, you should use `card: card` so that you get a card preview when you hover over the button. I get the feeling that @cryogen would prefer a card selection prompt here though.